opm/autodiff/BlackoilWellModel.hpp

```diff
@@ -358,7 +358,7 @@ namespace Opm {
             void computeRepRadiusPerfLength(const Grid& grid, Opm::DeferredLogger& deferred_logger);
 
 
-            void computeAverageFormationFactor(std::vector<double>& B_avg) const;
+            void computeAverageFormationFactor(std::vector<Scalar>& B_avg) const;
```

Comment:
I do not recall which function made me change the `double` to `Scalar` here. Hopefully it is not a bid deal.